contents/handbook/engineering/how-we-review.md

```diff
@@ -0,0 +1,42 @@
+---
+title: How we review PRs
+sidebar: Handbook
+showTitle: true
+---
+
+Almost all PRs made to repositories on PostHog will need a review from another engineer. We do this because almost every time we review a PR we find a bug, a performance issue, unnecessary code or UX that could have been confusing.
+
+## How to review
+
+1. Have a flick through the code.
+  - What to look for:
+    - Code that does the wrong thing or will produce bugs
+    - Code that you think will cause performance issues
+    - Are there tests for all of the new functionality, and do they test the right thing?
+    - Any security issues or project leakage?
```

Comment:
Q: What is project leakage?

Reply:
We've had a couple of quite serious bugs where we weren't filtering by team_id, potentially exposing customer data. We've had a ton more instances of the same problem that just lead to bad performance as we were doing full table scans when we didn't need to